cocos/platform/CCImage.cpp

```diff
@@ -1149,6 +1149,11 @@ bool Image::initWithPngData(const unsigned char * data, ssize_t dataLen)
         {
             premultipliedAlpha();
         }
+        else
+        {
+            _hasPremultipliedAlpha = true;
```

Comment:
Why enable `premultipliedAlpha` even png format isn't RGBA8888, any errors? maybe you use `CCImage` in wrong way.

Reply:
@drelaptop The new PR is #19138, you can check it.

Reply:
sorry, you hope `CCImage.hasPremultipliedAlpha()` return `true`, but you did `setPNGPremultipliedAlphaEnabled(false)` before. that is confusing.

> no need to new a PR for same bug fix, you can commit to same branch, then PR will auto update.

Reply:
@drelaptop `setPNGPremultipliedAlphaEnabled(false)` changes `PNG_PREMULTIPLIED_ALPHA_ENABLED` to false, it tells cocos2dx that pngs have been PMAed by developers, so cocos2dx needn't to do `premultipliedAlpha`, `premultipliedAlpha` takes about 40ms for 2048*2048 pngs.

Reply:
@drelaptop I read `CCImage.cpp`, and `setPNGPremultipliedAlphaEnabled(false)` maybe the only way to use PMAed pngs. 
Doesn't cocos2dx support  PMAed pngs?
![2](https://user-images.githubusercontent.com/12747983/47836450-4df4be80-dde3-11e8-889f-02d14978f8a6.jpg)